src/map_io/map_players_view_packet.cc

```diff
@@ -777,7 +777,7 @@ void MapPlayersViewPacket::read(FileSystem& fs,
 							   "\"%s\":%" PRIuS ": node (%i, %i) t = R: unexpected end of "
 							   "file while reading time_triangle_last_surveyed",
 							   plnum, survey_times_filename,
```

Comment:
Here's an additional fix you can do: the `plnum` variables need casting to `<unsigned int>`. They are `uint8_t`, which is interpreted as a `char` by C++, so we end up with garbage output.

`get_pos()` returns type `size_t`, so your changed casts are the correct approach.

Reply:
did that without completely understanding it ;)